challenges/objects-arrays.js

```diff
@@ -6,29 +6,68 @@
   object name, diet, weight, length, period
 */
 
+function Dinosaur (name, diet, weight, length, period){
+  this.name = name;
+  this.diet = diet;
+  this.weight = weight;
```

Comment:
Interesting, great to see you build a constructor for this here. I did as a student, although it ins't necessary it is great practice, and reinforces why we do them!